### DISCUSSION TITLE
Inovice Ninja

### DISCUSSION THREAD

**Discussion by andrasbacsai** [coollabsio/coolify#3295] [Service Template Requests]
https://github.com/invoiceninja/dockerfiles

**Comment by Strider27** [comment]
Current template might need to resolve fqdn to localhost for local PDF generation to work properly with snappdf (headless chromium), which in most cases should be used over hosted options.

```yaml
extra_hosts:
      - '$SERVICE_FQDN_INVOICENINJA:127.0.0.1'
```

Otherwise chromium times out when trying to adjust invoice design (for some when trying to generate invoices as well).
Looks like chromium times out trying to verify ssl if the above is not added, even if traefic is configured properly and is getting correctly signed certs from Let's Encrypt.
